python/mxnet/gluon/loss.py

```diff
@@ -194,6 +194,14 @@ class SigmoidBinaryCrossEntropyLoss(Loss):
         L = - \sum_i {label}_i * \log({pred}_i) +
             (1 - {label}_i) * \log(1 - {pred}_i)
 
+    A value `pos_weights > 1` decreases the false negative count, hence increasing
```

Comment:
please update the formulas above with the new `pos_weight` parameter. The documentation should specify how the formula changes when `from_sigmoid` is True and False and where in the formula `pos_weight` is used.